src/test-collection/formatters/list.ts

```diff
@@ -0,0 +1,33 @@
+import path from "node:path";
+import type { TestCollection, TestDisabled, FormatterListTest } from "..";
+
+export const format = (testCollection: TestCollection): FormatterListTest[] => {
```

Comment:
Here I once again dislike the fact that formatters are tightly coupled with TestCollection, even though conceptually a formatter should only know how to format a test and type-wise can be as simple as `(tests: Test[]) => FormattedTest[]`.

IMO it makes code simpler and more robust, reduces amount of types needed and reduces coupling

But that's my perspective and I do not insist on doing it this way, this may be done in the far future

Reply:
At the moment, the test reading method returns exactly the `TestCollection` structure and I don't want to force the user to convert `TestCollection` to tests array and then call a special method `format`. I mean it's not convenient right now. But I agree that `TestCollection` should be removed in next major in order to simplify the work with tests.